### DISCUSSION TITLE
[Bug] creating new package...how to add manifest?

### DISCUSSION THREAD

**Discussion by Chris25602** [ScoopInstaller/Scoop#5532] [Help and Questions]
## Bug Report
This probably me not knowing how scoop works but, I have searched and not found any answers. Please help me understand what I should be doing to create a new scoop package.

#### Current Behavior

[https://github.com/ScoopInstaller/Scoop/wiki/Creating-an-app-manifest](https://github.com/ScoopInstaller/Scoop/wiki/Creating-an-app-manifest)
Fork repo
git clone repo

`scoop create eks-anywhere https://github.com/Chris25602/eks-anywhere.git`
git add/commit/push

`scoop bucket add eks-anywhere https://github.com/Chris25602/eks-anywhere.git`
`scoop install eks-anywhere`
`Couldn't find manifest for ''.`
#### Expected Behavior

A new scoop package would be installed
#### Additional context/output

```
scoop bucket list

Name         Source                                             Updated             Manifests
----         ------                                             -------             ---------
eks-anywhere https://github.com/Chris25602/eks-anywhere.git     6/8/2023 7:05:04 PM         0
```
Also here is the manifest file generated by scoop create command.

`-a----          6/8/2023   7:56 PM            208 eks-anywhere.json`

How do I tell my scoop bucket there is a manifest?

#### Scoop Configuration

```
{
    "last_update":  "2023-06-08T17:43:00.1158154-05:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}

**Comment by HUMORCE** [comment]
- `https://github.com/Chris25602/eks-anywhere` is not a scoop bucket.
- EKS Anywhere does not support windows (no windows releases, binaries) ref. https://anywhere.eks.amazonaws.com/docs/getting-started/install/#administrative-machine-prerequisites
- `scoop create` is a useless feature for now, it can only create a base manifest based on the application's download link.
